No longer overwrite FITS configuration XML

mistydemeo opened the pull request:
In the past, the default FITS configuration XML was installed via its own package, and Archivematica would then overwrite it in its own install.

This custom configuration has been moved to our FITS package, and Archivematica no longer stomps over it. This should also ensure that FITS still works after Archivematica is uninstalled.

Commit:
No longer overwrite FITS configuration XML

In the past, the default FITS configuration XML was installed via its
own package, and Archivematica would then overwrite it in its own
install.

This custom configuration has been moved to our FITS package, and
Archivematica no longer stomps over it. This should also ensure that
FITS still works after Archivematica is uninstalled.

refs #6457

Hwesta commented:
Can/does this remove externals/fits/archivematicaConfigs/ directories?  Other than that, looks good.

mistydemeo commented:
This does - directories in git only exist insofar as files exist in them. Git doesn't track directories separately, so that directory structure will no longer exist as of this commit.

Commit:
MCPClient control: Alphabetize, lines < 80 cols

Commit:
MCPClient control: specify dep on FITS 0.8.0-3

Commit:
MCPClient control: specify dep on ffmpeg 2.1.0 or later

We will be pushing out an update to the ffmpeg script post-release
which will use ffv1 level 3, and that requires ffmpeg 2.1.0 or later.

Commit:
control: Depend on libavcodec-extra-55, not -53

-53 is an Ubuntu package that's associated with its older libav
release. It wouldn't actually be used by ffmpeg.
